PayPalNativePayments/src/main/java/com/paypal/android/paypalnativepayments/PayPalNativeShippingListener.kt

```diff
@@ -0,0 +1,29 @@
+package com.paypal.android.paypalnativepayments
+
+/**
+ * An optional listener to receive notifcations if the user changes their shipping information.
+ */
+interface PayPalNativeShippingListener {
+
+    /**
+     * Notify when the users selected shipping address changes
+     *
+     * @param actions actions to perform after patching the order
+     * @param shippingAddress the user's most recently selected shipping address
+     */
+    fun onPayPalNativeShippingAddressChange(
+        actions: PayPalNativeShippingActions,
+        shippingAddress: PayPalNativeShippingAddress
```

Comment:
Does this shippingAddressChange callback need `actions`? In stand we talked about only exposing it for shipping method changes.

Reply:
well that depends of what we want to expose to merchants. Do we `accept` every address change? or do we give the merchants the possiblity to reject and address change? (Maybe they dont operate in some states or something). Im fine either way

Reply:
Sure - we can leave it then. Maybe let's just specify in the docs that it is optional to use for shipping address but required to use for shipping method.